docs/source/commandline/commandline-all.rst

```diff
@@ -1807,7 +2872,18 @@ Core Sound Options
     Higher values increase audio delay but may help avoid buffer under-runs and
     audio interruptions.
 
-    The default is ``1``.
+    The default is ``1``. 
+
+    | PortAudio calculates audio_latency as 20ms steps.
+    | XAudio2 calculates audio_latency as 100ms steps.
+    | DSound calculates audio_latency as 10ms steps.
+    | CoreAudio calculates audio_latency as 40ms steps.
```

Comment:
Looking at the code:
```
    m_headroom = m_sample_bytes * (clamped_latency() * sample_rate() / 40);
    m_buffer_size = m_sample_bytes * std::max<uint32_t>(sample_rate() * (clamped_latency() + 3) / 40, 256U);
```
That’s dividing by 40, which gives 25ms per step.  Could you check that these numbers are actually correct for the others as well?

Reply:
I'm not familiar with the exact mechanics of that math; it was a best guesstimate at a quick glance for each. I'll take another look through all of it.